contrib/docker/Dockerfile

```diff
@@ -34,19 +30,19 @@ RUN adduser \
     --uid "${UID}" \
     "${USER}"
 
-# Build dcrd and other commands it provides
+# Build dcrd and other commands it provides.
 WORKDIR /go/src/github.com/decred/dcrd
 RUN git clone https://github.com/decred/dcrd . && \
     CGO_ENABLED=0 GOOS=linux \
     go install -trimpath -tags safe,netgo,timetzdata \
       -ldflags="-s -w" \
       . ./cmd/gencerts ./cmd/promptsecret
```

Comment:
Why bother building `gencerts` and `promptsecret`?

Reply:
My thought is that both of them are needed on a fairly regular basis.  `gencerts` is used if you want to use clientcert authentication (which I know a few people who do) and `promptsecret` is needed with `dcrctl` if you ever want to feed it a password.

Reply:
Ok so its "just in case". I thought I might have missed something and they are depended on by something.